### PR TITLE
Use shorter bugs and repository properties

### PR DESCRIPTION
**Pull request by zlatanvasovic**
Reference: https://docs.npmjs.com/files/package.json

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -72,9 +70,7 @@
   ],
   "license": "MIT",
   "main": "shrinkpack",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/JamieMason/shrinkpack.git"
+  "repository": "JamieMason/shrinkpack"
```
Comment (valscion) [inline package.json]: This line is missing a comma, causing `package.json` to be invalid and CI to fail because of that.

Comment (zlatanvasovic) [reply package.json]: Thanks, I hope it's fixed now.

File: package.json [hunk 2]
```diff
@@ -72,10 +70,7 @@
   ],
   "license": "MIT",
   "main": "shrinkpack",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/JamieMason/shrinkpack.git"
-  },
+  "repository": "JamieMason/shrinkpack",
```
Comment (valscion) [inline package.json:73]: https://docs.npmjs.com/files/package.json#repository

I was able to check that this holds also for npm@2, https://github.com/npm/npm/blob/v2.13.2/doc/files/package.json.md#repository

**Comment by valscion** [comment]
I wonder what was the reason why you think this is a good change? I don't have any strong opinions about this myself, just wondering what  improvements do you feel this brings?

**Comment by zlatanvasovic** [comment]
You have to understand that software development always requires changes. Every programmer tends to write something in a shorter, but more understandable way if that's possible. It's as simple as that.

There is no reason to keep something written in a incomprehensible, complicated way if the shorter and more understandable way is available.

**Comment by valscion** [comment]
I'm sorry if I came across rude, I was just interested in hearing your point why you wanted to send this PR. If I understood correctly, your proposition is that this makes the `package.json` "shorter and more understandable" — which I think is a good thing! :+1:

**Comment by zlatanvasovic** [comment]
@valscion It wasn't rude, don't worry. It's cool to discuss. :smile:

**Comment by JamieMason** [comment]
Works for me, happy to merge next time I pick up the project. Thanks a lot for contributing.

**Comment by JamieMason** [comment]
Merged into master 👍

**Pull request closed without merging**